SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -197,3 +198,31 @@ namespace SpatialConstants
 
 	const FString DEVELOPMENT_AUTH_PLAYER_ID = TEXT("Player Id");
 }
+
+FORCEINLINE Worker_ComponentId SchemaComponentTypeToWorkerComponentId(ESchemaComponentType SchemaType)
+{
+	switch (SchemaType)
+	{
+	case SCHEMA_CrossServerRPC:
+	{
+		return SpatialConstants::SERVER_RPC_ENDPOINT_COMPONENT_ID;
+	}
+	case SCHEMA_NetMulticastRPC:
+	{
+		return SpatialConstants::NETMULTICAST_RPCS_COMPONENT_ID;
+	}
+	case SCHEMA_ClientReliableRPC:
+	case SCHEMA_ClientUnreliableRPC:
+	{
+		return SpatialConstants::SERVER_RPC_ENDPOINT_COMPONENT_ID;
+	}
+	case SCHEMA_ServerReliableRPC:
+	case SCHEMA_ServerUnreliableRPC:
+	{
+		return SpatialConstants::CLIENT_RPC_ENDPOINT_COMPONENT_ID;
+	}
+	default:
```

Comment:
Not something for right now, but we should split this into multiple enums and stop checking default cases in these.

Reply:
Not sure I understand.

Reply:
The enum we are switching on has two distinct groups of elements that are unrelated. If we exhaustively check all possible values then we don't need these weird default cases. But we can't because this enum is odd. It's something to fix later. I just thought it while reading this.

Reply:
I see, is it ok to resolve this conversation then?